cypress/support/step_definitions/validations.md

```diff
@@ -28,18 +28,29 @@
 * `User validates lifecycle history for '1st party app' with 'background:foreground:background'`
 * `User validates lifecycle history for '3rd party app' with 'background:foreground'`
 
-## I '(start|stop)' performance metrics collection
+## Metrics collection process is '(initiated|stopped)'
 ### Purpose: To start or stop performance metrics service in device by passing appropriate intent to performance test handler
 
```

Comment:
query : why are we modifying performance glue code to handle metrics ?

Reply:
It is md file, just updated the glue like in document for performance  metrics